ok

```diff
@@ -0,0 +1 @@
+
```

Comment:
<resources>
    <!-- Default screen margins, per the Android Design guidelines. -->
    <dimen name="activity_horizontal_margin">16dp</dimen>
    <dimen name="activity_vertical_margin">16dp</dimen>
    <dimen name="textview_padding">8dp</dimen>
    <dimen name="textview_margin">8dp</dimen>
</resources>